src/test/test_db.py

```diff
@@ -2,11 +2,11 @@
 import unittest
 from unittest import TestCase
 from dotenv import load_dotenv
-from db import DBDriver, STATUS_OK, STATUS_RECEIPT_UNKNOWN_USER, STATUS_USER_ALREADY_EXIST, STATUS_RECEIPT_ALREADY_EXIST
+from src.db import DBDriver, STATUS_OK, STATUS_RECEIPT_UNKNOWN_USER, \
+    STATUS_USER_ALREADY_EXIST, STATUS_RECEIPT_ALREADY_EXIST, STATUS_USER_ALREADY_DEACTIVATED, STATUS_FAIL
 import random
 
-conf_pth = os.path.join(os.path.dirname(__file__), 'env')
```

Comment:
Need to be sure that local env file is used. If the script is called from another folder, that another file could be read.